mongodb/leafygreen-ui
# Updates Storybook build & config [LG-2992]

## ✍️ Proposed changes

Pre-work for: https://jira.mongodb.org/browse/LG-3025

- Updates `rollup.config.ts` to build storybook files as `esm` modules
- Updates `depcheck` script to ignore self-imports
- Updates Toast stories to import from built `@leafygreen-ui/toast` package
  - this ensures that the providers are the same when rendering the live-example on .design
- Patch bump for all packages (to ensure story file gets built)

## ✅ Checklist

### For bug fixes, new features & breaking changes


### For new components


## 🧪 How to test changes

1. Ensure `story.js` exists after running build
2. Link this PR to the associated .design branch https://github.com/mongodb/design/pull/246/files
3. Ensure live-examples are rendered on .design
